demo/apisix/conf/apisix.yaml

```diff
@@ -6,6 +6,82 @@ routes:
       nodes:
       type: roundrobin
     plugin_config_id: 1
+  -
+    id: jmap
+    uri: /jmap
```

Comment:
Could we specify `/oidc/jmap` (APISIX) -> `/jmap` (TMail), just like what we did with KrakenD?